install.sh

```diff
@@ -34,8 +36,16 @@ if ! (is_executable npm && is_executable node && is_executable git); then
   fi
 fi
 
-if [ -z $npm_needs_sudo ]; then
-  sudo npm install -g slap
+if [ -z "$npm_needs_sudo" ]; then
+  if [ -n '$needs_python_env' ]; then
```

Comment:
These single quotes won't work, as they're not interpolated.

This will also never be false as `$needs_python_env` is always nonempty.